micro-ui/web/micro-ui-internals/packages/modules/core/src/components/TopBarSideBar/SideBar/CitizenSideBar.js

```
@@ -163,14 +164,25 @@ export const CitizenSideBar = ({
   };
 
   const handleModuleClick = (url) => { 
-    url[0]==="/" ? 
+    
+    if(Digit.Utils.getMultiRootTenant()){
+      url=isEmployee?url.replace("/sandbox-ui/employee", `/sandbox-ui/${tenantId}/employee`): url.replace("/sandbox-ui/citizen", `/sandbox-ui/${tenantId}/citizen`)
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)